xpk.py

```diff
@@ -2490,17 +2783,49 @@ def workload_create(args) -> int:
   else:
     container = get_main_container(args, system, docker_image, resource_type)
 
-  yml_string = workload_create_yaml.format(args=args,
-                                           system=system,
-                                           container=container,
-                                           affinity=get_cpu_affinity(system.accelerator_type),
-                                           env=get_cpu_env(args.num_slices,system),
-                                           accelerator_label=create_accelerator_label(system.accelerator_type, system),
-                                           machine_label=create_machine_label(system.accelerator_type, system),
-                                           local_queue_name=_LOCAL_QUEUE_NAME)
-  tmp = write_temporary_file(yml_string)
-  command = f'kubectl apply -f {str(tmp.file.name)}'
-  return_code = run_command_with_updates(command, 'Creating Workload', args)
+  if args.use_pathways:
```

Comment:
within the `if args.use_pathways` code would be really good place for a helper function that exists in pathways_xpk_helpers.py.